draft-ietf-rtcweb-jsep.xml

```diff
@@ -1562,6 +1563,74 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           results in connectivity being established.</t>
         </section>
       </section>
+      <section title="RtpTransceiver" anchor="sec.transceiver">
+        <section title="stop" anchor="sec.transceiver-stop">
+          <t>
+            The stop method stops an RtpTransceiver.  This
+            will cause future calls to createOffer to generate a zero
+            port for the associated m= section.  See below for more
+            details.
+          </t>
+        </section>
+        <section title="isStopped" anchor="sec.transceiver-stopped">
+          <t>
+            The isStopped method returns "true" if the transceiver has
+            been stopped, either by a call to stopTransceiver or by
+            applying an answer that rejects the associated m= section, and
+            "false" otherwise.
+          </t>
+          <t>
+            A stopped RtpTransceiver does not send any
+            outgoing RTP or RTCP or process any incoming RTP
+            or RTCP.  It cannot be restarted.
+          </t>
+        </section>
+        <section title="setDirection" anchor="sec.transceiver-set-direction">
+          <t>
+            The setTransceiverDirection method sets the
+            direction of a transceiver, which affects the
+            direction attribute of the associated m= section on
+            future calls to createOffer and createAnswer.
+          </t>
+        </section>
+        <section title="setCodecPreferences" anchor="sec.transceiver-set-codec-preferences">
+          <t>
+            The setCodecPreferences method sets the codec
+            preferences of a transceiver, which affect the
+            presence and order of codecs of the associated
+            m= section on future calls to createOffer and
+            createAnswer.
+          </t>
+          <t>
+            The codec preferences of an RtpTransceiver can cause codecs
+            to be excluded by subsequent calls to createOffer and
+            createAnswer, in which case the corresponding media formats
+            in the associated m= section will be excluded. The codec
+            preferences cannot add media formats that would otherwise not
+            be present. This includes codecs that were not negotiated in a
+            previous offer/answer exchange that included the transceiver.
+          </t>
+          <t>
+            The codec preferences of an RtpTransceiver can also determine
+            the order of codecs in subsequent calls to createOffer and
+            createAnswer, in which case the order of the media formats in
+            the associated m= section will match. However, the codec
+            preferences cannot change the order of the media formats after
+            an answer containing the transceiver has been applied. At this
+            point, codecs can only be removed, not reordered.
+          </t>
+          <t>
+            Note that, as described in <xref target="RFC3264">
```

Comment:
This guidance is somewhat irrelevant here since we have a specific set of directives below indicating what should be sent. (i.e., the 'applying a remote description' section). 

So, you could either point at that, or omit this clause and say 'Note that the order of codecs in sCP does not affect which codec the implementation decides to send. It only affects ...'